config/zephyr/Kconfig

```diff
@@ -515,7 +524,7 @@ config CHIP_PW_RPC
 	bool "Pigweed RPC library"
 	default n
 	help
-	  Links the application with the libraries containing Pigweed RPC
+	  Links the application wCHIP_NXP_PLATFORMith the libraries containing Pigweed RPC
```

Comment:
sth wrong here

Reply:
Ah good catch. I must have pasted it accidentally while looking for occurrences in the project :)